packages/cloudflare/src/cli/build/patches/to-investigate/wrangler-deps.ts

```diff
@@ -107,18 +107,22 @@ function patchRequireReactDomServerEdge(config: Config) {
     const parameterName = p[0]!.getName();
     const bodyChildren = arrowFunction.getBody().getChildren();
     if (
-      bodyChildren.length !== 3 ||
-      bodyChildren[0]!.getFullText() !== "{" ||
-      bodyChildren[2]!.getFullText() !== "}"
+      !(
+        bodyChildren.length === 3 &&
+        bodyChildren[0]!.getFullText() === "{" &&
```

Comment:
I had approved your version, you could have kept it. 
Curious if you can remove the ! here and below now

Reply:
> I had approved your version, you could have kept it.

it's all good 🙂👍 


> Curious if you can remove the ! here and below now

No
![Screenshot 2025-01-08 at 17 48 17](https://github.com/user-attachments/assets/c52408df-9645-42c8-811a-92890b103096)